api/src/main/java/marquez/db/DatasetDao.java

```diff
@@ -194,6 +194,9 @@ default void setFields(Dataset ds) {
           + "ORDER BY d.name")
   List<Dataset> findAll(String namespaceName, int limit, int offset);
 
+  @SqlQuery("SELECT count(*) FROM datasets AS j WHERE j.namespace_name = :namespaceName")
+  int countFindAll(String namespaceName);
```

Comment:
I'd define the following methods for counts, as `findAll()` usually means that the method would return `n` rows:

```java
// total count
int count();

// total count by namespace
int countFor(namespaceName);
```

I would do that same for the `JobDao`

Reply:
Good idea!